package.json

```diff
@@ -2,10 +2,11 @@
   "name": "smile-identity-core",
   "version": "2.0.0",
   "description": "The official Smile Identity Node SDK",
-  "main": "index.js",
+  "main": "dist/index.js",
```

Comment:
should we add a "typings" definition next to this?

Reply:
I'm not sure how to add the typings when there are multiple declarations, will have to look it up
maybe @cmrfrd can assist with that

Reply:
Adding multiple declarations should be straight forward, it's very much similar to c/c++ headers.

https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html